<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSVAnalyzer</class>
 <widget class="QWidget" name="CSVAnalyzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSVAnalyzer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTableView" name="tableView"/>
         </item>
         <item>
          <widget class="QPushButton" name="chooseFileButton">
           <property name="text">
            <string>Choose file...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="analyseButton">
           <property name="text">
            <string>Analyse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="chooseNameProduct">
           <property name="placeholderText">
            <string>All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
